classes/phing/tasks/ext/phpunit/PHPUnitTask.php

```diff
@@ -482,7 +482,7 @@ protected function execute($suite)
             /**
              * Add some defaults to the PHPUnit filter
              */
-            $pwd = __DIR__;
+            $pwd = getcwd();
             $path = realpath($pwd . '/../../../');
```

Comment:
I think the main problem is the use of `realpath()` function inside a phar archive.
Could you please try with:
```php
$pwd = __DIR__;
$path = dirname($pwd, 3) . '/';
```